runtime/cudaq/platform/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
+include(HandleLLVMOptions)
```

Comment:
This was moved out of a CMakeLists.txt higher up in the tree and replicated in a lot of files. Did that fix a build problem?

Reply:
see my response above to another instance where you had commented regarding this.